tests/pytests/test_gc.py

```diff
@@ -420,15 +430,26 @@ def test_gc_oom(env):
     env.expect(config_cmd(), 'SET', 'FORK_GC_CLEAN_THRESHOLD', '0').ok()
     env.expect(config_cmd(), 'SET', 'FORK_GC_RUN_INTERVAL', '30000').ok()
     num_docs = 10
+    # Create index
+    env.expect('FT.CREATE', 'idx', 'SCHEMA', 't', 'TEXT').ok()
+    # Add some documents
     for i in range(num_docs):
         env.expect('HSET', f'doc{i}', 't', f'name{i}').equal(1)
-    env.expect('FT.CREATE', 'idx', 'SCHEMA', 't', 'TEXT').ok()
-    waitForIndex(env, 'idx')
+
+    # Get open file count
+    redis_server_pid = env.executeCommand('info')['process_id']
+    open_files_before_gc = get_open_file_count(redis_server_pid)
+    env.assertGreater(open_files_before_gc, 0)
+
+    set_tight_maxmemory_for_oom(env)
+
+    # Delete them all
     for i in range(num_docs):
         env.expect('DEL', f'doc{i}').equal(1)
-
-    env.expect('config', 'set', 'maxmemory', 1).ok()
```

Comment:
Is there a reason for this discrepancy? CC @lerman25

Reply:
I'm not sure, my bad that I didn't document it.
My guess is that set_tight/unlimited functions in common.py weren't backported to 2.X at the time.